compiler/rustc_hir_analysis/src/collect/item_bounds.rs

```diff
@@ -94,7 +98,15 @@ pub(super) fn explicit_item_bounds(
             kind: hir::ItemKind::OpaqueTy(hir::OpaqueTy { bounds, in_trait, .. }),
             span,
             ..
-        }) => opaque_type_bounds(tcx, def_id, bounds, *span, *in_trait),
+        }) => {
+            let substs = InternalSubsts::identity_for_item(tcx, def_id);
+            let item_ty = if *in_trait && rpitit_info.is_none() {
```

Comment:
Actually, why is this correct? Doesn't this mean that the item bounds for an RPITIT will contain opaque types, instead of projections?

Reply:
You're right, fixed.